Foxtrot/Tests/Properties/AssemblyInfo.cs

```diff
@@ -45,3 +46,5 @@
 // by using the '*' as shown below:
 [assembly: AssemblyVersion("1.0.0.0")]
 [assembly: AssemblyFileVersion("1.0.0.0")]
+
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true)]
```

Comment:
Just curious: why test parallelization is disabled? Maybe comment would be helpful if there is a reason for this...